src/words/environment/WordsObject.java

```diff
@@ -84,8 +108,16 @@ public void setProperty(String propertyName, Property property) throws WordsRunt
 
 		if (properties.containsKey(propertyName) && property.type == Property.PropertyType.NOTHING)
 			properties.remove(propertyName);
```

Comment:
If I remove a property on an object, and that property was referring to another object, then I'm no longer a referrer to that object, and that object's referrer list would need to remove me as a referrer, right?